### PR TITLE
Adds check to prevent false wall stacking, fixes #6449

### LINKED ISSUE #6449: You can stack false walls
You can build any amount of false walls on the same turf. Even if said turf is also an actual wall.

### PR DESCRIPTION
**Pull request by maartenberg**
(First PR, pls be gentle)
This should fix #6449, but it does not prevent placing a girder inside a false wall (and closing the false wall afterwards), only actually constructing the new (false) wall.

### CODE REVIEW

File: code/game/objects/structures/girders.dm [hunk 1]
```diff
@@ -89,6 +89,12 @@
 		if (!istype(src.loc, /turf/simulated/floor))
 			usr << "<span class='warning'>A floor must be present to build a false wall!</span>"
 			return
+		if (locate(/turf/simulated/wall) in src.loc)
```
Comment (Jordie0608) [inline code/game/objects/structures/girders.dm]: This should be a typecheck for the turf being a wall.

File: code/game/objects/structures/girders.dm [hunk 2]
```diff
@@ -89,6 +89,12 @@
 		if (!istype(src.loc, /turf/simulated/floor))
 			usr << "<span class='warning'>A floor must be present to build a false wall!</span>"
 			return
+		if (locate(/turf/simulated/wall) in src.loc)
+			usr << "<span class='warning'>There is already a wall present!</span>"
+			return
+		if (locate(/obj/structure/falsewall) in src.loc)
```
Comment (Jordie0608) [inline code/game/objects/structures/girders.dm]: This should look in contents rather than all of loc

**Comment by spasticVerbalizer** [comment]
The new commit works, do I need to squash?

File: code/game/objects/structures/girders.dm [hunk 3]
```diff
@@ -89,6 +89,12 @@
 		if (!istype(src.loc, /turf/simulated/floor))
 			usr << "<span class='warning'>A floor must be present to build a false wall!</span>"
 			return
+		if (istype(src.loc, /turf/simulated/wall))
+			usr << "<span class='warning'>There is already a wall present!</span>"
+			return
+		if (locate(/obj/structure/falsewall) in src.loc.contents)
+			usr << "<span class='warning'>There is already a false wall present!</span>"
```
Comment (Jordie0608) [inline code/game/objects/structures/girders.dm]: A bit of a nitpick that's not your fault but could you change these three uses of 'usr' to 'user', works the same but is a better way to handle it.

Comment (spasticVerbalizer) [inline code/game/objects/structures/girders.dm]: All right, I found two other uses of `usr` (line 33 and 203), I'll change those as well.

**Pull request merged**